hwdb.d/60-sensor.hwdb

```diff
@@ -394,7 +394,7 @@ sensor:modalias:acpi:KIOX000A*:dmi:bvnAmericanMegatrendsInternational,LLC.:bvrJP
  ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
 
 # T11 / Juno Tablet 2
-sensor:modalias:acpi:NSA2513*:dmi:*:svnDigitek:*
+sensor:modalias:acpi:NSA2513*:dmi:*
```

Comment:
Why do you drop the svn field? Please show the modalias you want to make the line matches.

Reply:
Digitek is hardcoded into the BIOS. So some tablet BIOS's dont have this. They might use svnDefaultstring instead. if so, it will not work.

Reply:
Sorry, I forgot to mention.

Instead it finds the acceleromenter Driver NSA2513 (we have just pushed this linux upstream).

Reply:
I mean, other tablet may use the same device with different config. So, we should add some marker to match the tablet. Please show the dmi modalias.

Reply:
Or, is the NSA (wow!) device setting should be universal?

Reply:
NSA is the Accelerometer's name. It will be part of da280 soon

https://github.com/torvalds/linux/blob/master/drivers/iio/accel/da280.c

Reply:
> I mean, other tablet may use the same device with different config. So, we should add some marker to match the tablet. Please show the dmi modalias.

I will get both modalias later today

Reply:
Here you can see both modalias. The only difference is the svn

$ cat /sys/class/dmi/id/modalias
dmi:bvnAmericanMegatrendsInternational,LLC.:bvr5.19:bd07/19/2023:br5.19:efr1.7:svnDefaultstring:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:skuDefaultstring:

$ cat /sys/class/dmi/id/modalias
dmi:bvnAmericanMegatrendsInternational,LLC.:bvr5.19:bd06/26/2023:br5.19:efr1.7:svnDigitek:pnDefaultstring:pvrDefaultstring:rvnDefaultstring:rnDefaultstring:rvrDefaultstring:cvnDefaultstring:ct3:cvrDefaultstring:skuDefaultstring:

Reply:
any updates?

Reply:
Sorry for late. Then, please list the following two modaliases:
```
sensor:modalias:acpi:NSA2513*:dmi:*:svnDigitek:*
sensor:modalias:acpi:NSA2513*:dmi:*:svnDefaultstring:*
```
Hopefully, other devices that have the same sensor has unique `svn` field.

Reply:
Updated. I can always add more modaliases later if anything changes.